i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/workflow/cue-actions.md

```diff
@@ -2,321 +2,539 @@
-### 用法示例
+**用法示例**
 
 ```
-apply: op.#ApplyRemaining & {
-  exceptions: ["applied-component-name"]
+import "vela/op"
+myApply: op.#ApplyInParallel & {
+  value: [{
+    kind: "Deployment"
+    apiVersion: "apps/v1"
+    metadata: name: "test-app"
+    spec: { 
+      replicas: 2
+      ...
+    }
+  }, {
+    kind: "Deployment"
+    apiVersion: "apps/v1"
+    metadata: name: "test-app2"
+    spec: { 
+      replicas: 2
+      ...
+    }
+  }]
 }
 ```
 
-## Slack
-
----
-
-向 Slack 发送消息通知。
+### Read
 
-### 操作参数
+在 Kubernetes 集群中读取资源。
 
-- url: Slack 的 Webhook 地址。
-- message: 需要发送的 Slack 消息，需要符合 [Slack 信息规范](https://api.slack.com/reference/messaging/payload) 。
+**参数定义**
 
 ```
-#Slack: {
-  url: string
-  message: {...}
+#Read: {
+	// +usage=The cluster to use
+	cluster: *"" | string
+	// +usage=The resource to read, this field will be filled with the resource read from the cluster after the action is executed
+	value?: {...}
+	...
 }
 ```
 
-### 用法示例
+**用法示例**
 
 ```
-apply: op.#Slack & {
-  url: webhook url
-  message:
-    text: Hello KubeVela
+import "vela/op"
+myRead: op.#Read & {
+  value: {
+    kind: "Deployment"
+    apiVersion: "apps/v1"
+    metadata: name: "test-app"
+  }
 }
 ```
 
-## DingTalk
+### List
+
+在 Kubernetes 集群中列出资源。
+
+**参数定义**
+
+```
+#List: {
+	// +usage=The cluster to use
+	cluster: *"" | string
+	// +usage=The resource to list
+	resource: {
+		// +usage=The api version of the resource
+		apiVersion: string
+		// +usage=The kind of the resource
+		kind: string
+	}
+	// +usage=The filter to list the resources
+	filter?: {
+		// +usage=The namespace to list the resources
+		namespace?: *"" | string
+		// +usage=The label selector to filter the resources
+		matchingLabels?: {...}
+	}
+	// +usage=The listed resources will be filled in this field after the action is executed
+	list?: {...}
+	...
+}
+```
 
----
+**用法示例**
 
-向钉钉发送消息通知。
+```
+import "vela/op"
+myList: op.#List & {
+  resource: {
+    kind: "Deployment"
+    apiVersion: "apps/v1"
+  }
+  filter: {
+    matchingLabels: {
+      "mylabel": "myvalue"
+    }
+  }
+}
+```
 
-### 操作参数
+### Delete
+
+在 Kubernetes 集群中删除资源。
+
+**参数定义**
+
+```
+
+#Delete: {
+	// +usage=The cluster to use
+	cluster: *"" | string
+	// +usage=The resource to delete
+	value: {
+		// +usage=The api version of the resource
+		apiVersion: string
+		// +usage=The kind of the resource
+		kind: string
+		// +usage=The metadata of the resource
+		metadata: {
+			// +usage=The name of the resource
+			name?: string
+			// +usage=The namespace of the resource
+			namespace: *"default" | string
+		}
+	}
+	// +usage=The filter to delete the resources
+	filter?: {
+		// +usage=The namespace to list the resources
+		namespace?: string
+		// +usage=The label selector to filter the resources
+		matchingLabels?: {...}
+	}
+}
+```
 
-- url: 钉钉的 Webhook 地址。
-- message: 需要发送的钉钉消息，需要符合 [钉钉信息规范](https://developers.dingtalk.com/document/robots/custom-robot-access/title-72m-8ag-pqw) 。
+**用法示例**
 
 ```
-#DingTalk: {
```

Comment:
Deprecated？

Reply:
It's just a wrap of  HTTPDo, I think this action can be deprecated in the near future to refactor the definition with op.#HTTPDo.